data/sqlutil/scanrow.go

```diff
@@ -60,7 +62,7 @@ func MakeScanRow(colTypes []*sql.ColumnType, colNames []string, converters ...Co
 	seen := map[string]int{}
 	for i, name := range colNames {
 		if j, ok := seen[name]; ok {
-			return nil, fmt.Errorf(`duplicate column names are not allowed, found identical name "%v" at column indices %v and %v`, name, j, i)
+			return nil, errorsource.DownstreamError(fmt.Errorf(`duplicate column names are not allowed, found identical name "%v" at column indices %v and %v`, name, j, i), false)
```

Comment:
Could we use  `DownstreamError` from `https://github.com/grafana/grafana-plugin-sdk-go/blob/97a7441690e9ddecbea56addfba03035097fabe8/backend/error_source.go#L76` instead of experimental.

Reply:
Hmm, interestingly when I replace it with backend.DownstreamError, it doesn't get marked as downstream. Only the string "downstream error" is appended (ignore the typo, was just making sure my replace works ): 
<img width="380" alt="Screenshot 2024-09-17 at 13 26 36" src="https://github.com/user-attachments/assets/477f75e8-f4dc-470a-bf7a-e1224d30c1bf"> Is this intended?

Reply:
Looks like this is a bug, reported here: https://github.com/grafana/grafana-plugin-sdk-go/issues/1090
We agreed we can merge this until that's fixed.